### DISCUSSION TITLE
Trying to use XBOX X Controller on a game that did not support it

### DISCUSSION THREAD

**Discussion by AM0T** [AntiMicroX/antimicrox#820] [Q&A]
Hello! I am trying to use XBOX X controller on Modern Warfare 3 but it will not work. Some other users said they got it to work buti cannot get it to work please help!

**Comment by Qzhor** [comment]
me too, only works in desktop

**Comment by BoomerangAide** [comment]
Have you tried running AntiMicroX in admin mode (compatibility options in the .exe properties for Win10) ?
